geometryAbstractPractice/src/test/java/comp/fundacionjala/geometryAbstractPractice/CircleTest.java

```diff
@@ -0,0 +1,30 @@
+package comp.fundacionjala.geometryAbstractPractice;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CircleTest {
+
+    private Circle circle;
+    private static final double perimeterExpected = 25.12;
```

Comment:
This variables are only used in the test method
So they should be defined in the test method.